src/Components/Subscriptions/index.js

```diff
@@ -1,9 +1,323 @@
-      <h2>Subscriptions</h2>
+      <section className={styles.list}>
+        <header className={styles.header}>
+          <h1 className={styles.title}>Subscriptions</h1>
+          <button
+            className={styles.btn}
+            onClick={() => {
+              setShowAdd(!showAdd);
+              setShowEdit(false);
+            }}
+          >
+            ADD Subs
+          </button>
+        </header>
+        <table className={styles.table}>
+          <tbody className={styles.tbody}>
+            <tr className={styles.tr}>
+              <th className={styles.thead}>Subscription</th>
+              <th className={styles.thead}>Activity</th>
+              <th className={styles.thead}>Member Last name</th>
+              <th className={styles.thead}>Date</th>
+            </tr>
+            {subscriptions.map((subscription) => (
+              <tr key={subscription._id} className={styles.tr}>
+                <td className={styles.td}>{subscription._id}</td>
+                <td className={styles.td}>{subscription.classId?.activityId?.name}</td>
+                <td className={styles.td}>{subscription.memberId?.lastName}</td>
+                <td className={styles.td}>{subscription.date.slice(0, 10)}</td>
+                <td className={styles.td}>
+                  <img
+                    src="/assets/images/edit-icon.svg"
+                    style={{ cursor: 'pointer' }}
+                    onClick={() => {
+                      setShowEdit(!showEdit);
+                      setShowAdd(false);
+                      setCurrentClassId(subscription.classId?._id);
+                      setCurrentMemberId(subscription.memberId?._id);
+                      setCurrentDate(subscription.date.slice(0, 10));
+                      setCurrentId(subscription._id);
+                    }}
+                  />
+                </td>
+                <td className={styles.td}>
+                  <img
+                    src="/assets/images/delete-icon.svg"
+                    style={{ cursor: 'pointer' }}
+                    onClick={() => {
+                      const shureDelete = confirm('Are you sure you want to delete?');
+                      if (shureDelete) {
+                        alert('Subscription deleted');
+                        onDelete(subscription._id);
+                      }
+                    }}
+                  />
+                </td>
+              </tr>
+            ))}
+          </tbody>
+        </table>
+        {showAdd && (
+          <form className={styles.form} onSubmit={onSubmit}>
+            <div className={styles.inputBox}>
+              <label className={styles.label}>Class ID:</label>
+              <select
+                type="text"
+                placeholder="class Id"
+                value={classId}
+                onChange={(e) => setClassId(e.target.value)}
+                required
+              >
+                <option value="">Seleccione una opción</option>
```

Comment:
Remove unused attribute

Reply:
Which? the value of "select an option"?. I should change it to english

Reply:
value

Reply:
> value

Yes! I fixed it and also corrected the language in the text !!